api/src/models/bcmi/permit-bcmi.js

```diff
@@ -0,0 +1,26 @@
+module.exports = require('../../utils/model-schema-generator')(
+  'PermitBCMI',
+  {
+    _schemaName: { type: String, default: 'PermitBCMI', index: true },
+    _sourceRefId:  { type: String, default: null, index: true },
+
+    read: [{ type: String, trim: true, default: 'sysadmin' }],
+    write: [{ type: String, trim: true, default: 'sysadmin' }],
+
+    mineGuid: { type: String, default: '' },
```

Comment:
Open question: do we need to include all of the other `Permit` fields here as well? Our other records always copy all of the master fields, regardless of whether or not they get shown on the frontends.  But maybe that isn't necessary? Not sure

Reply:
I was talking to @marklise about this and opted to follow what the others have done. However, I would think they should just live on the flavour.

Reply:
If I understand what you said correctly (possibly I don't), then we DO want to add all of the other permit fields here? `recordName`, `dateIssued`, `legislation`, etc.

Reply:
Oh sorry, I thought this was on the master. Umm, to me it seems unnecessary on the flavour, but maybe there was a reason it was done before. Do you think they should be? Would they be used for anything?

Reply:
> Do you think they should be? Would they be used for anything?

Not sure.  Trying to think if there is a good reason to include them, other than just because that is what was done in past. 

The only thing that comes to mind, is if we wanted to use those other values for filtering/searching. 
On LNG, we don't show that many fields, but some of them get used for searching/filtering.  BCMI in theory might be similar, though with collections, its harder to say, since now we aren't directly querying for bcmi records from the public site.

So not sure :/

I guess given that we have no current reason to change it, we can just leave it as is, and always migrate it over in future if it becomes necessary for w/e reason

Reply:
Okay, that sounds good.